llvm/lib/MCA/Stages/InOrderIssueStage.cpp

```diff
@@ -329,6 +335,17 @@ void InOrderIssueStage::updateCarriedOver() {
   else
     Bandwidth -= CarryOver;
 
+  // updateIssuedInst did not handle executed if issue had carry over.
+  if (CarriedOver.getInstruction()->isExecuted()) {
+    PRF.onInstructionExecuted(&IS);
+    LSU.onInstructionExecuted(IR);
+    notifyEvent<HWInstructionEvent>(
+        HWInstructionEvent(HWInstructionEvent::Executed, IR));
+    LLVM_DEBUG(dbgs() << "[E] Instruction #" << IR << " is executed\n");
```

Comment:
Could we use notifyInstructionExecuted here? I think it's doing the same thing

Reply:
Done!